File: circularlayoutmanager/src/main/java/com/kapil/circularlayoutmanager/OnItemClickListener.java
package com.kapil.circularlayoutmanager;

/**
 * This interface provides callback regarding recycler view item click and long click detected from
 * the scroll wheel when.
 */

public interface OnItemClickListener {

    void onItemClick(ScrollWheel scrollWheel, int childIndex);

    void onItemLongClick(ScrollWheel scrollWheel, int childIndex);
}

File: circular-layout-manager/src/main/java/com/kapil/circularlayoutmanager/OnItemClickListener.kt
package com.kapil.circularlayoutmanager

/**
 * This interface provides callback regarding recycler view item click and long click detected from
 * the scroll wheel when.
 */
interface OnItemClickListener {
    fun onItemClick(scrollWheel: ScrollWheel?, childIndex: Int)
    fun onItemLongClick(scrollWheel: ScrollWheel?, childIndex: Int)
}